ibis/expr/types.py

```diff
@@ -1158,6 +1183,22 @@ class DateColumn(ColumnExpr, DateValue):
     pass
 
 
+class TimeScalar(ScalarExpr, TimeValue):
+    @property
+    def _factory(self):
```

Comment:
I don't think you need this since there's no metadata attached to the time type.